File: Leetcode/PY/Leetcode1446-连续字符串.py
class Solution:
    def maxPower(self, s: str) -> int:
        i,j,res = 0,0,0
        while i < len(s):
            while j < len(s) and s[i] == s[j]: j += 1
            res = max(res,j - i)
            i = j
        return res

File: Leetcode/Rust/Leetcode1446-连续字符串.rs
impl Solution {
    pub fn max_power(s: String) -> i32 {
        let (mut res,mut i,mut j) = (0,0,0);
        let s = s.as_bytes();
        while i < s.len() as i32{
            while j < s.len() as i32 && s[i as usize] == s[j as usize]{
                j += 1;
            }
            res = res.max(j - i);
            i = j;
        }
        res
    }
}